packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1435,10 +1435,13 @@ export abstract class Chart extends Observable implements AgChartInstance {
             this.zoomManager.updateZoom();
         }
 
-        const miniChartInstance = navigatorModule?.miniChartInstance;
-        if (miniChartInstance != null) {
-            const seriesStatus = this.applySeries(miniChartInstance, processedOptions);
-            this.applyAxes(miniChartInstance, processedOptions, seriesStatus);
+        const navigator = this.modules.get('navigator') as any;
```

Comment:
notice you already have the `navigatorModule` 8 lines up.

Reply:
Updated